CONTRIBUTING.md

```diff
@@ -51,7 +51,7 @@ git clone --recurse-submodules https://github.com/determined-ai/determined.git
   - python3-venv
   - python3-wheel
   - python3-dev
-- Node (>= 16.13, < 20)
+- Node (>= 18.16.0, < 19)
```

Comment:
shouldn't we keep the max version to 20?

Reply:
i dont think its a good idea because node v19 will be EOL in 3 weeks https://endoflife.date/nodejs

Reply:
and the max version was up v19, but i think we can support v18 and v20?

Reply:
Ah, I see.